pkg/node/volumemgr.go

```diff
@@ -649,15 +645,11 @@ func (m *VolumeManager) discoverVolumeCRs(freeDrives []*drivecrd.Drive) error {
 				}
 			}
 
-			partUUID = bdev.Children[0].PartUUID
-			if partUUID == "" {
-				partUUID = uuid.New().String() // just generate random and exclude drive
-				ll.Warnf("There is no part UUID for partition from device %v, UUID has been generated %s", bdev, partUUID)
-			}
+			volUUID := uuid.New().String() // just generate random and exclude drive
 
-			volumeCR := m.k8sClient.ConstructVolumeCR(partUUID, api.Volume{
+			volumeCR := m.k8sClient.ConstructVolumeCR(volUUID, api.Volume{
 				NodeId:       m.nodeID,
-				Id:           partUUID,
+				Id:           volUUID,
```

Comment:
In that case we can't determine for which partition this Volume CR was created. We do not work with multiple partitions right now but in future we might implement it. Volume CR should unambiguously indicate particular partition on particular drive. For that next should be true: `partUUID == VolumeCR.Spec.Id`